tools/optimize/dx12_optimize_util.cpp

```diff
@@ -71,9 +71,10 @@ void CreateResourceValueTrackingConsumer(
     application = std::make_shared<gfxrecon::application::Application>(app_string, file_processor);
     application->InitializeDx12WsiContext();
 
-    // Use default replay options, except dcp.
+    // Use default replay options, except dcp and gpu index.
     decode::DxReplayOptions dx_replay_options;
     dx_replay_options.discard_cached_psos = true;
+    dx_replay_options.override_gpu_index  = options.override_gpu_index;
```

Comment:
There are merge conflicts so this PR needs to be rebased onto the latest `dev` and the conflicts resolved. PR #1207 replaced the var `DxReplayOptions::discard_cached_psos` with `DxReplayOptions::use_cached_psos` so that may be the source of the conflict.